### PR TITLE
New patreron supporter

### PR DESCRIPTION
**Pull request by acrymble**
Adds Laurissa Stokes as an "Educator" level Patreon supporter to our various /supporters pages

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

File: es/lecciones/transformacion-datos-xml-xsl.md [hunk 1]
```diff
@@ -514,7 +514,6 @@ Para eliminar la sangría del texto, necesitarás hacerte cargo directo del espa
 ## Bibliografía recomendada por el traductor
 
 - Hunter, David *et al*. *Beginning XML*. Indianapolis: Wiley Publishing, 2007 (cuarta edición). Impreso.
-- Riley, Jenn. [*Understanding Metadata: What is Metadata, and What is For?*](https://www.niso.org/publications/understanding-metadata-2017) NISO, 2017. Web.
```
Comment (anisa-hawes) [inline es/lecciones/transformacion-datos-xml-xsl.md:517]: Hello @acrymble. You could consider replacing the link with this one, which directs to the article as a PDF: https://groups.niso.org/apps/group_public/download.php/17446/Understanding%20Metadata.pdf

or this one, https://www.niso.org/node/12366, which directs to the abstract.

What do you think?

**Review by anisa-hawes — COMMENTED** [review]
I have suggested some alternatives to the broken link for Jenn Riley's article.

File: es/lecciones/transformacion-datos-xml-xsl.md (hunk 1 above)
Comment (acrymble) [reply es/lecciones/transformacion-datos-xml-xsl.md:517]: Good suggestion. I've made the change.

**Review by anisa-hawes — APPROVED** [review]
Super! It's good to be able to keep that citation in!

**Pull request merged**